jffs2/libjffs2.c

```diff
@@ -596,11 +596,6 @@ static int libjffs2_create(void *info, oid_t *dir, const char *name, oid_t *oid,
 				dev_find_oid(p->devs, dev, d_inode(dentry)->i_ino, 1);
 			break;
 		case otSymlink:
-			/* empty target check */
-			if (dentry->d_name.len >= namelen) {
```

Comment:
This change seems somewhat not related. Why is it needed?

Reply:
https://github.com/phoenix-rtos/phoenix-rtos-filesystems/blob/19cb5240dc4377c7167fb9301ac52192747ecdac/jffs2/libjffs2.c#L562

This condition makes no sense. While NAND has been tested the symlinks couldn't be created.